comfy_extras/nodes_mask.py

```diff
@@ -5,7 +5,19 @@
 
 from nodes import MAX_RESOLUTION
 
+def tensor_to_rgb(img):
+    # convert from bw to rgb : cwh -> bchw
```

Comment:
I don't think this is correct. I'm fairly certain that masks in ComfyUI are `BHW` and images are `BHWC`. Your logic here works out when the batch size is 1, but will break if there is a batch of more than one mask.

Reply:
Ah, I see -- `.movedim(-1, 1)` is called before calling this function for the RGB case -- so it would be `BCHW`. Still, I think you still want to insert the channel dimension rather than the batch dimension.

Reply:
> Ah, I see -- `.movedim(-1, 1)` is called before calling this function for the RGB case -- so it would be `BCHW`. Still, I think you want to insert the channel dimension rather than the batch dimension.

The batch dimension is needed because it accesses channel dimension via index position, its expected at index[1] in the composite node so CHW would also fail. I think this is because it accepts batched images, and will repeat the mask for each image in the batch size, but atleast in my experience even if its a single image, most core nodes want BCHW, even though there are move dims happening.